CRP.Mvc/Controllers/SearchController.cs

```diff
@@ -54,6 +54,34 @@ public ActionResult Index(string searchTerm)
 
 
             return View(viewModel);
+        }
+
+        public ActionResult Index(string searchTerm)
+        {
+            var viewModel = SearchViewModel.Create(Repository);
+
+            if (!string.IsNullOrEmpty(searchTerm))
+            {
+                var itemList = _searchTermProvider.GetByTerm(searchTerm).Where(a => a.Private == false && a.Available && a.Expiration != null && a.Expiration >= DateTime.UtcNow.ToPacificTime().AddDays(-15)).AsQueryable(); //.OrderByDescending(a => a.Expiration >= DateTime.UtcNow.ToPacificTime()).ThenBy(a => a.Expiration);
+                var unexpiredItems = itemList
+                    .Where(a => a.Available && !a.Private && a.Expiration != null && a.Expiration >= DateTime.UtcNow.ToPacificTime().Date)
+                    .OrderBy(a => a.Expiration)
+                    .ToList();
+
+                var expiredItems = itemList
+                    .Where(a => a.Available && !a.Private && a.Expiration != null && a.Expiration >= DateTime.UtcNow.ToPacificTime().AddDays(-15).Date && a.Expiration < DateTime.UtcNow.ToPacificTime().Date)
```

Comment:
@jSylvestre probably an edge case but you should do the date math first and then convert timezones, on you'll occasionally get a value that is off.  `DateTime.UtcNow.ToPacificTime().AddDays(-15)` should be `DateTime.UtcNow.AddDays(-15).ToPacificTime()`